generic3g/actions/CMakeLists.txt

```diff
@@ -10,4 +10,7 @@ target_sources(MAPL.generic3g PRIVATE
   ConvertUnitsAction.F90
 
   TimeInterpolateAction.F90
+  AccumulatorAction.F90
+  MeanAccumulator.F90
+  MaxAccumulator.F90
```

Comment:
What about MinAccumulator.F90 - the file is there but not compiled for now?

Reply:
Done